src/algebra/group_with_zero.lean

```diff
@@ -34,6 +34,89 @@ and require `0⁻¹ = 0`.
 
 set_option old_structure_cmd true
 
+variables {G₀ : Type*}
+
+mk_simp_attribute field_simps "The simpset `field_simps` is used by the tactic `field_simp` to
```

Comment:
Now that this attribute is available in this file, should a bunch of lemmas in this file also be added to the simpset?

Reply:
Besides that, LGTM

Reply:
This attribute was available in this file before this PR because this file imported `algebra.ring`. Do we manually add `@[field_simps]` to lemmas from this file somewhere?

Reply:
Ooh, then I misunderstood.